### PR TITLE
OSDOCS-2173: Added new NICs for 4.9/4.10

### PR DESCRIPTION
**Pull request by ctauchen**
[**UPDATE 2021-11-17**] This PR was merged early, reverted, and recreated in a new PR: https://github.com/openshift/openshift-docs/pull/38878

https://issues.redhat.com/browse/OSDOCS-2173

Applies to `enterprise-4.9` and `enterprise-4.10`

See #38632 for release note.

Preview: https://deploy-preview-38380--osdocs.netlify.app/openshift-enterprise/latest/networking/hardware_networks/about-sriov#supported-devices_about-sriov

SME: @zshi-redhat 
QE: @zhaozhanqi

### CODE REVIEW

File: modules/nw-sriov-supported-devices.adoc [hunk 1]
```diff
@@ -27,6 +37,21 @@
 |8086
 |158b
 
+|Intel
+|E810-CQDA2
```
Comment (zshi-redhat) [inline modules/nw-sriov-supported-devices.adoc:41]: Please also add `E810-2CQDA2` which has the same Vendor and Device IDs with `E810-CQDA2`.

Comment (ctauchen) [reply modules/nw-sriov-supported-devices.adoc:41]: In #35174, we consolidated variants of the same device when the vendor and device ID matched. Is there something different about `E810-2CQDA2` that means it should have a separate listing?

Comment (zshi-redhat) [reply modules/nw-sriov-supported-devices.adoc:41]: Looking at the product pages, they are almost the same, except some minor differences in the bracket height and Data Rate Per Port.

E810-CQDA2: https://ark.intel.com/content/www/us/en/ark/products/192558/intel-ethernet-network-adapter-e810cqda2.html
E810-2CQDA2: https://ark.intel.com/content/www/us/en/ark/products/210969/intel-ethernet-network-adapter-e8102cqda2.html

Let me double check.

Comment (braultatgithub) [reply modules/nw-sriov-supported-devices.adoc:41]: also side by side comparison for reference:
https://ark.intel.com/content/www/us/en/ark/compare.html?productIds=210969,192558

**Comment by zhaozhanqi** [comment]
LGTM except adding the `E810-2CQDA2`

**Comment by jboxman-rh** [comment]
@ctauchen epic!

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Pull request merged**